use_freespace_pointcloud: false
tsdf_voxels_per_side: 16
voxel_carving_enabled: true
mesh_color_mode: "color"
use_const_weight: false
use_tf_transforms: true
method: "fast"
enable_icp: false
# topology server config
world_frame: "world"
clear_distant_blocks: true
dense_representation_radius_m: 8.0
# gvd integration and graph extraction
min_diff_m: 1.0e-3
min_weight: 1.0e-6
positive_distance_only: true
parent_derived_distance: true
min_basis_for_extraction: 2
extract_graph: true
voronoi_config:
    mode: "L1_THEN_ANGLE"
    min_distance_m: 0.30
    parent_l1_separation: 10
    parent_cos_angle_separation: 0.1
graph_extractor:
    min_extra_basis: 2
    min_vertex_basis: 4
    merge_new_nodes: true
    node_merge_distance_m: 0.5
    edge_splitting_merge_nodes: true
    max_edge_split_iterations: 5
    max_edge_deviation: 4
    add_freespace_edges: true
    freespace_active_neighborhood_hops: 1
    freespace_edge_num_neighbors: 3
    freespace_edge_min_clearance_m: 0.05
    add_component_connection_edges: true
    connected_component_window: 10
    connected_component_hops: 2
    component_nodes_to_check: 50
    component_nearest_neighbors: 1
    component_max_edge_length_m: 5.0
    component_min_clearance_m: 0.05
    remove_isolated_nodes: true
